cost/google/unutilized_ip_addresses/google_unutilized_ip_addresses.pt

```diff
@@ -0,0 +1,193 @@
+name "Google Unutilized IP Addresses"
+rs_pt_ver 20180301
+type "policy"
+short_description "Checks Google for Unutilized IP Addresses. See the [README](https://github.com/rightscale/policy_templates/tree/master/cost/google/unutilized_ip_addresses/) and [docs.rightscale.com/policies](http://docs.rightscale.com/policies/) to learn more."
```

Comment:
That's a really long short description and a really short long description. 😕

Reply:
Seems like we should have a better standard that uses these fields as intended, especially since these policies also serve as examples for customers.